roles/sysbench/templates/sysbench.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Should we template this out more (considering we could add additional workloads using sysbench?).  We pass 
```
test:cpu
```
We could add a `if test == cpu` in the template

Reply:
All the basic tests (CPU, Memory) etc unless you go for a database workload(in which case you need to create a database first) have the same format sysbench --test=<name> params.  So it doesn't look like it is needed now?

Reply:
Actually you are right..might need to change it.